File: Auto-generated-tasks/Difficulty-1/Task-5/Solution.go
package "main"
import "fmt"
NN:= 5
h:= 0
Rez:= 0
var T [NN]float64
for h = 0; h < NN; h++ {
	if (T[h] < 11)
		Rez = Rez + 1
}
fmt.PrintLn("Rez=", Rez)

File: Auto-generated-tasks/Difficulty-1/Task-5/Solution.cs
int NN = 5, h, Rez = 0;
double[] T = new double[NN];
for (h = 0; h < NN; h++) {
	if (T[h] < 11)
		Rez = Rez + 1;
}
Console.WriteLine("Rez=" + Rez);